lib/raster3d/test/test.raster3d.lib.html

```diff
@@ -7,8 +7,3 @@ <h2>DESCRIPTION</h2>
 <h2>AUTHOR</h2>
 
 S&ouml;ren Gebbert
```

Comment:
Author name should be Soeren Gebbert.

Reply:
I am not sure, he wrote his name like that.

Reply:
Then it's okay.